### ISSUE TITLE
Unable to load URL

### ISSUE THREAD

**Issue by taktikal17** [pmatsinopoulos/instagram_scraping#1]
Hi @pmatsinopoulos,

Thank you for the script. 
I would like to use it, unfortunately I got an error when instagram should be launched.
I uninstalled capybara/capybara-webkit/site_prism before installing them again.
I still got this error..

Thanks!
Have a good one!

`Using proxy: 
Email: MyUserName
Hash Tag: MyHashtag
Please, give me your password to login to Instagram: 
MyPassword
Comments to be used: "Dope"
/usr/local/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.7.1/lib/capybara/webkit/browser.rb:305:in`check': Unable to load URL: https://instagram.com/ because of error loading https://instagram.com/: Unknown error (Capybara::Webkit::InvalidResponseError)
    from /usr/local/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.7.1/lib/capybara/webkit/browser.rb:211:in `command'
    from /usr/local/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.7.1/lib/capybara/webkit/browser.rb:19:in`visit'
    from /usr/local/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.7.1/lib/capybara/webkit/driver.rb:48:in `visit'
    from /usr/local/lib/ruby/gems/2.3.0/gems/capybara-2.5.0/lib/capybara/session.rb:232:in`visit'
    from /usr/local/lib/ruby/gems/2.3.0/gems/capybara-2.5.0/lib/capybara/dsl.rb:51:in `block (2 levels) in <module:DSL>'
    from /usr/local/lib/ruby/gems/2.3.0/gems/site_prism-2.7/lib/site_prism/page.rb:17:in`load'
    from instagram_scraping.rb:88:in `<main>'`

<img width="537" alt="screenshot at feb 11 15-53-18" src="https://cloud.githubusercontent.com/assets/8819239/12979656/ddf98c42-d0d8-11e5-8388-dceb0d2a65ab.png">

**Comment by pmatsinopoulos** [comment]
So, are you trying to run another script? Not the `instagram_scraping.rb` ?

**Comment by taktikal17** [comment]
It is the instagram_scraping.rb 

I just delete the part:

`def read_from_file(filename)
  full_path = "#{File.expand_path("..", __FILE__)}/#{filename}"
  result = []
  File.readlines(full_path).each do |line|
    line.chomp!
    result << line unless line.empty?
  end
  result
end`

I got an error with the file path. I change the comment for:
`comments = ARGV[2].chomp`

It was to move forward to test the entire script. This function was not called after.

The rest of the script it entirely yours: instagram_scraping.rb

(the screenshot was top show the line 88)

Thanks!

**Comment by pmatsinopoulos** [comment]
Have you set up the environment variables for PROXY as required by the program? If yes, then I believe that it may be problem with the proxy that you are using. You can remove/comment out the lines that configure Capybara to use proxy: https://github.com/pmatsinopoulos/instagram_scraping/blob/master/instagram_scraping.rb#L44-L49

When you can also change the line:

https://github.com/pmatsinopoulos/instagram_scraping/blob/master/instagram_scraping.rb#L93

and set it to 

   Capybara.default_driver = :selenium

This, if everything is ok, will open the Firefox (if you have it installed). Does it open it?

If it opens, does it load the Instagram page?

I will spend some time on that later on to see what's the problem. But it is an old program and I have not maintain it since the first time I made it work. Will come back to you.

**Comment by pmatsinopoulos** [comment]
I have updated the program. Instagram has changed their UI, hence I had to update it. Now it works, I have tested it that. **But...**
1. It does not seem to be working with `:webkit` driver. So, I have changed the default to be `:selenium`. See line: https://github.com/pmatsinopoulos/instagram_scraping/blob/master/instagram_scraping.rb#L93
2. If you do not have proxy, comment out the lines that configure Capybara to use proxy.
3. I have updated the README file to be more accurate about how to use the program with comments.

@taktikal17 Let me know whether the above is ok so that I can close the issue.
